source/air/src/parser.rs

```diff
@@ -94,9 +94,9 @@ impl Parser {
             Node::List(nodes) if nodes.len() > 0 => {
                 match &nodes[..] {
                     [Node::Atom(s), Node::List(labels), e] if s.to_string() == "location" => {
-                        let spans = self.nodes_to_spans(labels)?;
+                        let spans = self.nodes_to_labels(labels)?;
                         let expr = self.node_to_expr(e)?;
-                        return Ok(Arc::new(ExprX::LabeledAssertion(spans, expr)));
+                        return Ok(Arc::new(ExprX::LabeledAxiom(spans, expr)));
```

Comment:
The printer has cases for both `LabeledAxiom` and `LabeledAssertion`, but I only see `LabeledAxiom` in the parser.  We should be able to round-trip between the printer and parser.  Should we have separate syntax, like `axiom_location` and `location` to distinguish these?